sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubsTestEnvironment.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using Azure.Messaging.EventHubs.Core;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   Represents the ambient environment in which the test suite is
+    ///   being run, offering access to information such as environment
+    ///   variables.
+    /// </summary>
+    ///
+    public class EventHubsTestEnvironment: TestEnvironment
```

Comment:
Why change the class name?  Since this is scoped to only Event Hubs tests, I'd prefer that we keep it `TestEnvironment` and just fully qualify the base class in this file.   Am I overlooking something that would prevent that from working properly?

Reply:
Technically it should compile but I think InteliSense would be always confused and show two types with the same name.